README.md

```diff
@@ -1,48 +1,62 @@
+
+* IPsec recipe is validated only on Intel&reg; IPU E2100 target
+* Refer to https://ipdk.io/documentation/Recipes/InlineIPsec/
+* YANG model for IPsec SAD: https://github.com/ipdk-io/openconfig-public/blob/master/release/models/ipsec/openconfig-ipsec-offload.yang
+* Reference P4 program to enable IPsec on DPDK target: https://github.com/ipdk-io/networking-recipe/tree/main/p4src/Inline_IPsec
+
+strongSwan Integration with Inline Crypto Engine
 ========================================================
-1. The user should start and stop the strongSwan services as described in strongSwan manual.
-2. The StrongSwan configuration files need to be configure by user as described in 
+* The user should start and stop the strongSwan services as described in strongSwan manual.
+* The strongSwan configuration files must be configured by user as described in 
 			https://wiki.strongswan.org/projects/strongswan/wiki/UserDocumentation
-3. Make sure the infrap4d (openconfig, p4runtime) is running before we start strongSwan services.
-4. StrongSwan hosts (Host-A<---->Host-B) should be connected to negotiate the keys.
-5. The user should use the best practices followed by opensource community for other control plane related configurations.
+* Confirm that the networking-recipe (the `infrap4d` process, which acts as the gRPC server) is running before starting strongSwan services.
+* strongSwan hosts (Host-A<---->Host-B) should be connected to negotiate the keys.
+* The user should follow best practices followed by opensource community for other control plane related configurations.
  
-Build and run StrongSwan on Linux
-=====================================================================================================================
-	#Clone this repo and build dependency using scripts.
-	source env_setup_acc.sh
-	./install_dep_packages_acc.sh
+Getting Started
+===================
+
+Clone this repository and install the dependencies
+
+```bash
+git clone https://github.com/ipdk-io/ipsec-recipe.git
+cd ipsec-recipe
+source env_setup_acc.sh
+./install_dep_packages_acc.sh
+```
+
+Clone strongSwan and switch to version 5.9.3 (Any version can be used but verification has been completed on v5.9.3)
+
+```bash
+cd ipsec_offload_plugin/
+git clone  https://github.com/strongswan/strongswan.git
+cd strongswan/
+git checkout tags/5.9.3 -b 5.9.3
+cd -
+```
+
+Build strongSwan
 
-	cd ipsec_offload_plugin/
-	#clone Strongswan and Switch to version 5.9.3 (Any version can be used but verified on 5.9.3)
-	git clone  https://github.com/strongswan/strongswan.git
-	cd strongswan/
-	git checkout tags/5.9.3 -b 5.9.3
-	cd ..
+```bash
+./swanbuild_p4.sh -t native -o $DEPS_INSTALL --enable_grpc
```

Comment:
@vinintel what is `$DEPS_INSTALL` here? Need some info to describe this

Reply:
Its path to installed dependent packages, exported as part of "source env_setup_acc.sh" command